pandas/core/generic.py

```diff
@@ -3718,7 +3718,9 @@ def tail(self, n=5):
     def sample(self, n=None, frac=None, replace=False, weights=None,
                random_state=None, axis=None):
         """
-        Returns a random sample of items from an axis of object.
+        Return a random sample of items from an axis of object.
+
+        You can use `random state` for reproducibility
```

Comment:
I think since this is specifically discussing a parameter, it can be moved to the "Parameters" section. And we can mention reproducible in the examples.

Reply:
can you address this